src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -92,6 +95,12 @@ public StreamingResponseBody streamEvents(
 
         return outputStream -> {
 
+            if  (floodService.isConsumptionBlocked(eventTypeName)) {
```

Comment:
@adyach I can't really find where you block by application. Or am I missing something?